unit/test/Get-VSTeamInfo.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+Set-StrictMode -Version Latest
+
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path).Replace(".Tests.", ".")
+
+. "$here/../../Source/Classes/VSTeamVersions.ps1"
+. "$here/../../Source/Classes/VSTeamProjectCache.ps1"
+. "$here/../../Source/Private/common.ps1"
+. "$here/../../Source/Public/$sut"
+
+Describe 'Get-VSTeamInfo' {
+   # Mock the call to Get-Projects by the dynamic parameter for ProjectName
+   Mock Invoke-RestMethod { return @() } -ParameterFilter {
+      $Uri -like "*_apis/projects*"
+   }
+
+   . "$PSScriptRoot\mocks\mockProjectDynamicParamMandatoryFalse.ps1"
+
+   Context 'Get-VSTeamInfo' {
+      AfterAll {
```

Comment:
Just being curious: What is AfterAll doing in Pester?

Reply:
It is a clean up function. After all the tests it is run.